### ISSUE TITLE
Create bespoke DRS for data to be processed by ESMValTool

### ISSUE THREAD

**Issue by ehogan** [MetOffice/CMEW#116]
CDDS will write files to a directory structure that does not exactly match the BADC Data Reference Syntax (DRS). Therefore, for CMEW, a bespoke DRS will need to be used. This can be achieved by using the ESMValTool [developer configuration file](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/quickstart/configure.html#developer-configuration-file).

The location of the standardised data (within the installed workflow) will need to be added to the ESMValTool user configuration file.

**Acceptance criteria**:

- **Given** the path to the standardised data 
- **When** the `configure_process` task is run
- **Then** the ESMValTool user configuration is updated to capture the `rootdir` and `drs` settings required to read the standardised model data
- **And** that user configuration file can be used for a successful run of ESMValTool (may require a new issue on modifying developer config file)

**Details**:

- The path to the standardised data will be provided by `configure_standardise` before the data is actually standardised.
- Depends on #137.
- Estimate: medium, requires discussion.

**Planning**

- Step 1: Investigate CEDA tool "restructure-for-cmip6", see #154.
- Step 2: TBD, since it will depend on whether step 1 identifies any serious issues, and may involve exploring alternative options.
